apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/plugin/loader/InterceptorInstanceLoader.java

```diff
@@ -80,6 +78,9 @@
             }
             if (inst != null) {
                 INSTANCE_CACHE.put(instanceKey, inst);
+                logger.debug("interceptor instance is created for class '{}', instanceKey is: {}", className, instanceKey);
```

Comment:
What is the purpose of adding a log for this? I think even most agent developer doesn't care much about this.

Reply:
I don't know whether we need that. I am OK to discuss. Right now, there are only @ascrutae @candyleer and I have experiences and developed core codes. The opinion of mine is, most people even don't understand what the agent really does.

Reply:
I've remove the logger just now. we can add this later in sometime when more people wants to learn more about the detail work what the agent does.

Reply:
> I've remove the logger just now. we can add this later in sometime when more people want to learn more about the detail work what the agent does.